src/FontInterface.php

```diff
@@ -0,0 +1,72 @@
+<?php declare(strict_types=1);
+
+namespace HelloFresh\Ausraster;
+
+interface FontInterface
```

Comment:
I think this interface is too big, if we apply the Interface Segregation principle it makes it more flexible. 
In general I see here at least `FontTypeInterface`, `FontEmphasisInterface` or `FontStyleInterface`